Source/Core/Core/NetPlayProto.h

```diff
@@ -80,6 +80,7 @@ struct NetSettings
   bool m_SyncCodes;
   std::string m_SaveDataRegion;
   bool m_SyncAllWiiSaves;
+  int m_WiimoteExtension[4];
```

Comment:
```suggestion
  std::array<int, 4> m_WiimoteExtension;
```

Reply:
Should I change the memcard array too for consistency?